suite-common/wallet-config/src/utils.ts

```diff
@@ -48,6 +48,14 @@ export const normalizeNetworkAccounts = (network: Network): NormalizedNetworkAcc
 export const isBlockbookBasedNetwork = (symbol: NetworkSymbol) =>
     networks[symbol]?.backendTypes.some(backend => backend === 'blockbook');
 
+// TODO: move to networksConfig
```

Comment:
I will extend `backendType` in networks config or find a smart way. However, it does not make sense to do it now after freeze